data/test/scenarios/test_add_or_sub_in_leadership.cfg

```diff
@@ -73,7 +86,7 @@
             [/filter]
             variable=b
         [/store_unit]
-        {ASSERT ({VARIABLE_CONDITIONAL b.hitpoints equals 25})}
+        {ASSERT ({VARIABLE_CONDITIONAL b.hitpoints equals 79})}
```

Comment:
To confirm then, the final damage taken by the defender here is 21, which is calculated by:
* 5 damage (Elvish Archer's base damage)
* +12 (from [damage])
* +1.25 damage (from leadership_no_cum)
* +1.25 damage (from leadership_pos)
* +1.25 damage (from leadership_cum)
This adds up to 20.75, which rounds to 21. Correct?

And this is expected, since even though leadership_no_cum has `cumulative=no`, it's still cumulative with other [leadership] abilities that don't share that ID?

Reply:
@Pentarctagon test modified